src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -135,7 +203,10 @@ class PaymentInformation extends React.Component {
         <>
           <div className="vet360-profile-field">
             <ProfileFieldHeading
-              onEditClick={directDepositIsSetUp && this.props.editModalToggled}
+              editTarget={EDIT_TARGETS.bankName}
```

Comment:
Instead of passing `editTarget` and then pulling it off the event later, you could change `onEditClick` to:

```js
onEditClick={directDepositIsSetUp && () => this.handleEditClick(EDIT_TARGETS.bankName)
```

(This assumes you also make the change I suggested above to be able to get rid of `bind`)

Reply:
Still need to do the child-component-prop roundtrip here, but I'm now calling handleEditClick with new signature per your suggestion (`= e => { ... }`).

Tried your arrow-function change, but babel's eslint-parser chokes on all variations of this boolean-plus-arrow-function jsx-attribute.  "A bridge too far"?  :D